### ISSUE TITLE
UI for saving URL is slow to load

### ISSUE THREAD

**Issue by autonome** [mariofink/epic-pinboard#8]
When I click the icon, the panel pops up and then there's a beat... and then the UI loads into the panel.

I wanted to use Epic Pinboard to avoid that bit of lag which the bookmarklet has, so it would a nice improvement :)

**Comment by mariofink** [comment]
Good point! The delay is only caused by loading the suggested tags. Loading those tags async would make the UI a lot snappier. Will have a look into this. Thanks for the suggestion.

**Comment by maxkolenko** [comment]
Then it would be great to have an option to disable the suggested tags feature. I guess most people have their own tag nomenclature.
